bin/balena-dev

```diff
@@ -30,3 +45,30 @@ require('ts-node').register({
 	transpileOnly: true,
 });
 require('../lib/app').run();
+
+// Modify package.json oclif paths from build/ -> lib/, or vice versa
+function modifyOclifPaths(revert) {
+	const fs = require('fs');
+	const packageJsonPath = path.join(rootDir, 'package.json');
```

Comment:
This line was modified from your original commit for two reasons:
* `balena-lint` complained about a variable named `path` shadowing the globally imported `path` module.
* Before this change, it was necessary to `cd` to the cloned repo (`balena-cli`) root dir before executing `balena-dev`. After this change, `balena-dev` may be executed from any folder, e.g.:
```
cd myApp
~/balena-cli/bin/balena-dev build .
```